mixlib-install.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.files         = %w{LICENSE Gemfile Rakefile} + Dir.glob("*.gemspec") + Dir.glob("{bin,lib,support}/**/*")
   spec.executables   = ["mixlib-install"]
   spec.require_paths = ["lib"]
-
+  spec.required_ruby_version = ">= 2.6.0"
```

Comment:
I  _think_ we decided not to force newer versions so people can use this on like an old Mac to get the latest chef... we wanted to go "soft" min requirement.